projects/bp-gallery/src/components/views/exhibitions/exhibition-helper.ts

```diff
@@ -0,0 +1,29 @@
+import { useCreateExhibitionPictureMutation } from '../../../graphql/APIConnector';
+import { FlatPicture } from '../../../types/additionalFlatTypes';
+import { channelFactory } from '../../../helpers/channel-helpers';
+import { MutationFunction } from '../../../types/helper';
+
+export const addExhibitionPicture = async (
+  createExhibitionPicture: MutationFunction<typeof useCreateExhibitionPictureMutation>,
+  exhibitionId: string,
+  selectedPictures: FlatPicture[]
+) => {
+  if (!exhibitionId) return;
+  const exhibitionBroadcast = channelFactory(`exhibition-${exhibitionId}`);
```

Comment:
this channel never gets closed again, even when it is no longer used. Please look at the presentation-channel.hook.ts that implemented this pattern with useEffect